src/plugins/data_source/server/types.ts

```diff
@@ -17,7 +17,7 @@ import {
 } from '../common/data_sources';
 
 import { CryptographyServiceSetup } from './cryptography_service';
-import { DataSourceError } from './lib/error';
+import { DataSourceError } from '../common/data_sources';
```

Comment:
Thanks for this change.
Let's double check all files import DataSourceError from the updated path,